pkg/rtc/participant_internal_test.go

```diff
@@ -247,7 +247,7 @@ func TestMuteSetting(t *testing.T) {
 func TestConnectionQuality(t *testing.T) {
 	testPublishedTrack := func(loss, numPublishing, numRegistered uint32) *typesfakes.FakePublishedTrack {
 		t := &typesfakes.FakePublishedTrack{}
-		t.PublishLossPercentageReturns(loss)
+		t.GetConnectionScoreReturns(float64(loss))
```

Comment:
these tests don't appear useful anymore with this change. It appears to be testing averaging of connection scores between audio and video. That's useful, but perhaps a different way to get there?

It would be good to test the `GetConnectionScore` functions somehow, so we know they are performing the right computation given different scenarios.